build.zig

```diff
@@ -38,6 +38,7 @@ pub fn build(b: *Builder) void {
     b.makePath(kern_path.toSlice()) catch unreachable;
     b.makePath(a_path.toSlice()) catch unreachable;
 
+    src_files.append("config") catch unreachable;
```

Comment:
I'm going to be wary as before we had the issue of double functions and variables, As config.zig doesn't use any other functions this shouldn't be a problem. But once someone does, this could cause the same issue. So may need to come up with a better solution for adding additional files.
You do import this file into kmain.zig, so does this need to be here?

Reply:
That's a good point. There were some reasons to do with build.zig why I added the extra file but with some extra experimentation I've realised how to overcome them. I'll remove config.zig and import the options directly into kmain.zig.